sdk-dotnet/src/LittleHorseSDK/LittleHorse.Common/Exceptions/LHApiException.cs

```diff
@@ -0,0 +1,25 @@
+﻿using LittleHorseSDK.Common.proto;
```

Comment:
Do we still need this after the grpc refactor @sauljabin ? Currently, the other SDK's just throw the unadulterated GRPC exceptions.